src/Aspire.Hosting.Azure.AppService/AzureAppServiceInfrastructure.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Lifecycle;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Aspire.Hosting.Azure.AppService;
+
+internal sealed class AzureAppServiceInfrastructure(
```

Comment:
Maybe we should have  LifecycleHook/Hook suffix on this type?

Reply:
All of the types we have today are called this:
- https://github.com/dotnet/aspire/blob/e08e36e778f2caf14ff48da39d8b918bbccfa40b/src/Aspire.Hosting.Kubernetes/KubernetesInfrastructure.cs
- https://github.com/dotnet/aspire/blob/e08e36e778f2caf14ff48da39d8b918bbccfa40b/src/Aspire.Hosting.Docker/DockerComposeInfrastructure.cs
- https://github.com/dotnet/aspire/blob/e08e36e778f2caf14ff48da39d8b918bbccfa40b/src/Aspire.Hosting.Azure.AppContainers/AzureContainerAppsInfrastructure.cs